docs/src/APIs/Bucket.md

```diff
@@ -12,6 +12,17 @@ ClimaLand.Bucket.PrescribedSurfaceAlbedo
 ClimaLand.Bucket.BucketModel
 ```
 
+```@docs
```

Comment:
Could we add the second method too? https://github.com/CliMA/ClimaLand.jl/blob/32e96955b837ff4ec609f99553212abd19f9371a/src/standalone/Bucket/Bucket.jl#L265-L336